.github/workflows/e2e_lighthouse_performance_acceptance_tests.yml

```diff
@@ -312,6 +312,7 @@ jobs:
           suite:
             - blog-admin-tests/assign-roles-to-users-and-change-tag-properties
             - blog-editor-tests/try-to-publish-a-duplicate-blog-post-and-get-blocked
+            - exploration-editor-tests/load-complete-and-restart-exploration
```

Comment:
Not sure about the name -- you're mainly testing the preview functionality, right? So, perhaps, `preview-exploration` ?

Reply:
Yes, the tests are primarily testing the preview functionality. However, I’m curious if the preview-exploration includes restarting of the exploration? For now, I have updated it to load-complete-and-restart-exploration-preview.
Looking for your suggestion.

Reply:
I think what you have is probably fine. Thanks!

Reply:
sure, thanks.